src/Symfony/Component/Config/Tests/Builder/GeneratedConfigTest.php

```diff
@@ -86,10 +86,11 @@ public function testConfig(string $name, string $alias)
         $expectedOutput = include $basePath.$name.'.output.php';
         $expectedCode = $basePath.$name;
 
-        // to regenerate snapshot files, uncomment these lines
-        // (new Filesystem())->remove($expectedCode);
-        // $this->generateConfigBuilder('Symfony\\Component\\Config\\Tests\\Builder\\Fixtures\\'.$name, $expectedCode);
-        // $this->markTestIncomplete('Re-comment the line above and relaunch the tests');
+        if ($_SERVER['TEST_GENERATE_FIXTURES'] ?? false) {
```

Comment:
for consistency (or getenv() everywhere?) :
```suggestion
        if ($_ENV['TEST_GENERATE_FIXTURES'] ?? false) {
```